server/src/server.ts

```diff
@@ -16,22 +16,6 @@ import setUpMongo from "../mongo/mongoDbManager";
 
   setUpMongo();
 
-  app.use(cookieParser());
-  
-  app.use(cors({
```

Comment:
you are right that it shouldn't be like that, but we still need cors

Reply:
yoga add these cors by default, i dont think we need more cors then that but maybe i miss something so let me know if more cors needed
![image](https://user-images.githubusercontent.com/62710926/235793217-b19e763a-f012-4d22-8836-ceb9d9c94ae0.png)